Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -327,5 +318,34 @@ public class HitBehaviorDestroyOnSelect : MonoBehaviour
 }
 ```
 
+### Example #4: Use hand rays and eye gaze input together!
+Hand rays take priority over head and eye gaze targeting. This means, if hand rays are enabled, the moment the hands come into view, the hand ray will act as the primary pointer. 
+However, there might be situations in which you want to use hand rays while still detecting whether a user is looking at a certain hologram. Easy! Essentially you require two steps:
+
+**1. Enable the hand ray:** 
+To enable the hand ray, go to _Mixed Reality Toolkit -> Input -> Pointers_. 
+In the _EyeTrackingDemo-00-RootScene_ where the Mixed Reality Toolkit is configured once for all of the eye tracking demo scenes, you should see the _EyeTrackingDemoPointerProfile_. 
+You can either create a new _Input Profile_ from scratch or adapt the current eye tracking one:
+- **From scratch:** 
+In the _Pointers_ tab, select the _DefaultMixedRealityInputPointerProfile_ from the context menu. 
+This is the default pointer profile that already has the hand ray enabled! 
+To change the default cursor (an opaque white dot), simply clone the profile and create your own custom pointer profile. 
+Then replace _DefaultCursor_ with _EyeGazeCursor_ under _Gaze Cursor Prefab_.  
+    
```

Comment:
Consider adding a profile that has these changes, and just pointing people to it directly (i.e. EyeGazeInputPointerProfile.asset)

Reply:
Not in this change, which we can keep scoped to docs - just a thing to think about

---
In reply to: [308433456](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5432#discussion_r308433456) [](ancestors = 308433456)

Reply:
Yeah, sounds good! I'll keep it in mind. Thanks, Will! There might be some other changes coming up with the integration with the MRTK scene hub anyways.